presto-tests/src/main/java/io/prestosql/tests/statistics/MetricComparison.java

```diff
@@ -57,9 +57,7 @@ public Result result(MetricComparisonStrategy metricComparisonStrategy)
         if (!estimatedValue.isPresent()) {
             return NO_ESTIMATE;
         }
-        if (!actualValue.isPresent()) {
-            return NO_BASELINE;
-        }
+        checkState(actualValue.isPresent(), "baseline estimate is not present");
```

Comment:
s/"baseline estimate"/"actual value" ?

i think it's not "estimate", it's regular `count` or something